src/blueprints/common/index.js

```diff
@@ -125,10 +125,21 @@ export default {
       }
     },
     async done (context) {
-      if (!this.nuxt.options.dev) {
+      if (!context.rootOptions.dev) {
```

Comment:
Whats actually supposed to be the difference between rootOptions.dev and nuxt.options.dev? Cause atm they are the same

Reply:
I thought we were supposed to always use the context object now?

Reply:
That applies to the use of eg `this.$press`. Nuxt options should still always be the same for any blueprint root or blueprint config.